nx-X11/programs/Xserver/hw/nxagent/NXcomposite.c

```diff
@@ -1,359 +0,0 @@
-#ifndef NXAGENT_UPGRADE
```

Comment:
Actually, disregard my previous comment. Deleting this file is probably okay. It's not referenced in `nx-X11/programs/Xserver/hw/nxagent/Imakefile` and non-empty if `NXAGENT_UPGRADE` is not set, which is not the case with recent NX.